docs/qtpromise/qpromise/delay.md

```diff
@@ -20,3 +20,32 @@ auto output = input.delay(2000).then([](int res) {
     // called 2 seconds after `input` is fulfilled
 });
 ```
+
+*Since: 0.6.0*
+
+```cpp
+QPromise<T>::delay(std::chrono::milliseconds msec) -> QPromise<T>
+```
+
+This is a convenience overload accepting durations from the the standard C++ library. It is available only if your compiler supports the `<chrono>` header. 
```

Comment:
OK, thanks, I was not sure about the docs, I just typed them in in VS Code. I'll check with the npm. Is there a specific node.js version requirement for it?

Reply:
I think Node.js >= 8.6 since it's the version required by [VuePress](https://vuepress.vuejs.org/), but I would install the [latest LTS (12.15)](https://nodejs.org/en/download/). If you are not familiar with Node.js projects, you need to run:

```sh
npm install
npm run docs:lint
```